features/rni.feature

```diff
@@ -117,6 +117,9 @@ Functionaliteit: RNI-deelnemer voor geleverde gegevens
       Dan heeft de response een persoon met de volgende 'naam' gegevens
       | naam      | waarde |
       | voornamen | Peter  |
+      En heeft de persoon de volgende 'verblijfplaats' gegevens
+      | naam | waarde                 |
+      | type | VerblijfplaatsOnbekend |
```

Comment:
@JohanBoer de verblijfplaats heeft land met een waarde (ongelijk 0000), dus het type is VerblijfplaatsBuitenland.

Naar mijn mening moet de gba API in dit geval land meeleveren en dus ook bijbehorende rni.
De proxy gebruikt land om het type te bepalen en kan niet meer bepalen dat die rni niet geleverd zou moeten worden.
Ook kan je zeggen dat omdat type wordt geleverd, er iets gedaan is met het rni gegeven en moet dus juist wel rni geleverd worden.

Dus:
1. dit scenario is zinvol, maar hoort bij een andere rule
2. er moet een ander scenario komen die de rule wel illustreert

Reply:
Ik snap niet zo goed waarom " land" geleverd zou moeten worden als daar in Fields niet om gevraagd is. Ik denk wel dat het type "VerblijfplaatsBuitenland" zou moeten zijn. 

De GBA-API levert de RNI al niet mee als er uit het feit dat er geen gegevens uit verblijfplaats geleverd worden (behalve het type) kan de proxy opmaken dat rni niet geleverd hoeft te worden. 

Of het leveren van het type beschouwd moet worden als het leveren van veblijfplaatsgegevens is denk ik een uitspraak van de PO. Dat zou wel betekenen dat als er rni is bij de verblijfplaats dat deze ALTIJD geleverd wordt omdat wij polyformisme toepassen. 

@CathyDingemanse : Wil jij uitsluitsel geven ? Moet RNI meegeleverd worden als alleen het type van de verblijfplaats geleverd wordt ?

Reply:
Aangepast zoals besproken